### DISCUSSION TITLE
Welcome to Success-with-Nvidia-545-drivers-in-Ubuntu-22.04-LTS Discussions!

### DISCUSSION THREAD

**Discussion by RocketCityDynamics** [RocketCityDynamics/Success-with-Nvidia-545-drivers-in-Ubuntu-22.04-LTS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GG3834ZF** [comment]
Hello and grats for this very precise post. Ever tried it on a RTX2060 ? I assume it should work, and I'll let you know in case it does as soon as I dare trying it ;)
